src/commands/bgrewriteaof.json

```diff
@@ -0,0 +1,19 @@
+{
+    "BGREWRITEAOF": {
+        "summary": "Asynchronously rewrite the append-only file",
+        "complexity": "PATCH__TBD__55__",
```

Comment:
in theory the command itself doesn't block redis's main thread for long, so it's O(1).
but also, maybe it's ok to skip this field or leave it empty for some commands?

Reply:
i will make it optional